src/renderRepoCard.js

```diff
@@ -88,19 +97,21 @@ const renderRepoCard = (repo, options = {}) => {
       <style>
       .header { font: 600 18px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${titleColor} }
       .description { font: 400 13px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${textColor} }
+      .template { font: 400 13px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${textColor}; font-weight: 500 }
       .gray { font: 400 12px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${textColor} }
       .icon { fill: ${iconColor} }
-      .archive-badge { font: 600 12px 'Segoe UI', Ubuntu, Sans-Serif; }
+      .archive-badge, .template-badge { font: 600 12px 'Segoe UI', Ubuntu, Sans-Serif; }
```

Comment:
No we should just use one class called "badge" and use that on both

Reply:
done